packages/react-dom-bindings/src/events/ReactDOMEventListener.js

```diff
@@ -345,6 +345,7 @@ export function getEventPriority(domEventName: DOMEventName): EventPriority {
     case 'select':
     case 'selectstart':
       return DiscreteEventPriority;
+    case 'beforetoggle':
```

Comment:
It's a continuous event for now to match `toggle`. But I feel like both should be discrete events. From how I understood @gaearon, "discrete" can be seen as "intentional events" and I feel like this applies to toggle since it either requires a click or pressing of enter.

Probably best to change the priority in a follow-up though to make it easier to bisect potential perf regressions.

Reply:
agreed this should be discrete

Reply:
I'll do this in a follow-up so that we can bisect later.

Reply:
Fixed in https://github.com/facebook/react/pull/29176